installer/dbscript/dbscript-2.sql

```diff
@@ -4263,7 +4263,10 @@ VALUES ('apiv2_time_time_sheet_config', 'API-v2 Time - Time Sheet Start Day Conf
        ('api_v2_time_copy_activities', 'API-v2 Time - Project Copy activities', 1, 1, 0, 0),
        ('apiv2_time_employee_timesheets', 'API-v2 Time - Employee Timesheets', 1, 1, 1, 0),
        ('apiv2_time_employee_timesheet_list', 'API-v2 Time - Employee Timesheet List', 1, 0, 0, 0),
-       ('apiv2_time_default_timesheet', 'API-v2 Time - Default Timesheet', 1, 0, 0, 0);
+       ('apiv2_time_default_timesheet', 'API-v2 Time - Default Timesheet', 1, 0, 0, 0),
+       ('apiv2_time_reports', 'API-v2 Time - Reports', 1, 0, 0, 0),
+       ('apiv2_time_reports_data', 'API-v2 Time - Reports Data', 1, 0, 0, 0),
+       ('time_report_project_report', 'Time - Project report', 1, 0, 0, 0);
```

Comment:
Seems we don't need this data group, already there is a data group for this. `time_project_reports`

Reply:
done